package.json

```diff
@@ -56,6 +56,6 @@
     "test-on-travis": "yarn lint && yarn build && nyc ava --concurrency=2 --timeout=2m"
   },
   "workspaces": [
-    "packages/!(rule-amp-validator)"
+    "packages/*"
```

Comment:
@sarvaje Does `rule-amp-validator` now work?

Reply:
it is working on windows

Reply:
Not working for me :/

Reply:
Even if we remove the `rule-amp-validator` folder we will have problems because rules have a `devDependency` on `sonarwhal: 0.24.0`. The only solution I've found for now in Windows machines is to use bash.
We should revert @sarvaje's change to continue ignoring `rule-amp-validator` so once we publish the first version we can still use the regular prompt.

Reply:
I don't remember how I did it, but `rule-amp-validator` works perfect in my machine using the windows cmd :s

Reply:
> I don't remember how I did it,

@sarvaje Did you use magic again?

Reply:
>@sarvaje Did you use magic again? 😛

Hahahahaha it looks like that xD

Reply:
So, just to double check before do anything. I have to keep this `"packages/!(rule-amp-validator)"`, right?